pvlib/tracking.py

```diff
@@ -302,6 +302,14 @@ def singleaxis(apparent_zenith, apparent_azimuth,
         between the tracking axes has a gcr of 2/6=0.333. If gcr is not
         provided, a gcr of 2/7 is default. gcr must be <=1.
 
+    side_slope : float, default 0.0
+        The slope in degrees of the "system plane" perpendicular to the
+        tracker axes. The "system plane" is defined as the plane that
+        contains all of the tracker axes. EG north-south trackers on a
+        3-degree eastern slope would have a 3-degree side slope, depending
+        on the tracker axis azimuth. Use ``calc_system_tracker_side_slope``
+        for more complicated system planes.
```

Comment:
Should we switch it to conform with convention? IE positive if east for NS with 180&deg; azimuth? I think we'd only have to change the sign in one place, here from plus: 
[`temp = np.clip(axes_distance*cosd(wid + side_slope), -1, 1)`](../../../mikofski/pvlib-python/blob/24e19f2ae1cd250551b99910cc05ebb7d9e9d5af/pvlib/tracking.py#L441)
to minus
`temp = np.clip(axes_distance*cosd(wid - side_slope), -1, 1)`

Reply:
So thinking about this more, I'd rather not change it because someone will always be unhappy. If they're used to thinking of  morning tracker angle as negative, and afternoon as positive when tracker azimuth is facing 180&deg; how will they feel about a tracker in the southern hemisphere facing north? Would they be okay with the convention being reversed, or would they expect us to keep using azimuth of 180&deg; and then a negative zenith? I hope something like this table will be enough:


| tracker azimuth | pos side slope | neg side slope | pos axis tilt | neg axis tilt |
|----------------|----------------|----------------|------------|-------------|
| 0 (north)           | west                  | east                   | north           | south           |
| 180 (south)       | east                  | west                   | south          | north           |